hash_practice/exercises.py

```diff
@@ -1,19 +1,52 @@
 
+def create_anagram_key(string):
+    return ''.join(sorted(string))
+
+
 def grouped_anagrams(strings):
     """ This method will return an array of arrays.
         Each subarray will have strings which are anagrams of each other
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(n*m)

```

Comment:
I'm going to assume that `n` is the number of words in the list and that by `m` you mean the number of letters in each word. That being said, note that the call to `sorted` will actually take O(m * log(m)) time, making this more accurately O(n * m * log(m)).

However, we can make a simplifying assumption since we know that the input list is made up of English words. English words tend not to get long, only about 5 letters per word on average, so the effect of `m` is going to be dwarfed by the effect of `n`, since there can easily be hundreds or thousands of words in the list. With that assumption, we can just say the time complexity is just O(n).